specification/cognitiveservices/OpenAI.Inference/models/translation.create.tsp

```diff
@@ -0,0 +1,45 @@
+import "@typespec/rest";
+import "@typespec/http";
+
+import "./transcription.create.tsp";
+
+using TypeSpec.Rest;
+using TypeSpec.Http;
+
+namespace Azure.OpenAI;
+
+@doc("Translation request.")
+model TranslationOptions {
+
+    @doc("The audio file to translate. Binary data.")
+    // how do we handle binary format for a member?
+    file: string;
+
+    @doc("An optional text to guide the model's style or continue a previous audio segment. The prompt should be in English.")
+    prompt?: string;
+
+    @doc("The format of the transcription output, in one of these options: json, text, srt, verbose_json, or vtt.")
+    @projectedName("json", "response_format")
+    responseFormat?: TranscriptionFormat = TranscriptionFormat.json;
+
+    @doc("The sampling temperature, between 0 and 1. Higher values like 0.8 will make the output more random, while lower values like 0.2 will make it more focused and deterministic. If set to 0, the model will use log probability to automatically increase the temperature until certain thresholds are hit.")
+    temperature?: float32 = 0;
+}
+
+@doc("Translation response.")
+model Translation {
+    @doc("Translation task.")
+    task: string;
+
+    @doc("Translated text.")
+    text: string;
+
+    @doc("Language.")
+    language: string;
+
+    @doc("Duration.")
+    duration: uint32;
```

Comment:
I believe there are preferred types (with encoding attributes, as needed) for offset/durations, that'll in turn reflect to language-appropriate types like TimeSpan where appropriate.

Reply:
There is a `duration` type for duration types :) [doc link](https://microsoft.github.io/typespec/language-basics/built-in-types#date-and-time-types)

Reply:
changed and annotated with `@encode` as suggested on a different comment.